server/__init__.py

```diff
@@ -1,9 +1,18 @@
 from flask import Flask
+from server.models import sqldb
+from apscheduler.schedulers.background import BackgroundScheduler
 import os
 import redis
 
 app = Flask(__name__)
 
+# sql
+app.config['SQLALCHEMY_DATABASE_URI'] = os.environ.get("DATABASE_URL", "sqlite:///:memory:")
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+sqldb.init_app(app)
+with app.app_context():
+    sqldb.create_all()
```

Comment:
Is this call idempotent? I think it's supposed to be called only on first run.

Reply:
The `sqldb.create_all()` call doesn't do anything if the tables already exist. I could separate this code into a database creation script, but that would result in an extra step when setting up the server.